src/main/java/bookup/SQLHelper.java

```diff
@@ -1,6 +1,87 @@
 package bookup;
+import javax.xml.transform.Result;
+import java.sql.*;
+import java.util.*;
 
 /** Provides helper methods for interacting with the database */
 class SQLHelper {
-	
+
+    /** Adds specified book to database */
+    static void add(Book book) {
+        Connection con = connect();
+        try {
+            Statement statement = con.createStatement();
+            String SQLAddStatement = String.format("insert into books values(%s, '%s', '%s', %s, %s, '%s');", book.getISBN(), book.getTitle(), book.getAuthor(),book.getPages(), book.getYear(), book.getImageURL());
+            statement.executeUpdate(SQLAddStatement);
+            con.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
so if something went wrong, we will just silently ignore? maybe throw a new exception

Reply:
For example?

Reply:
Example of something going wrong? Could be anything. Example of an exception to throw? For now just a new RuntimeException